### PR TITLE
holochain-rust: version bump to v0.0.45-alpha1

### PR DESCRIPTION
**Pull request by peeech**
Bumps version of holochain-rust to v0.0.45-alpha1, also bumps all DNAs to a compatible version

### CODE REVIEW

**Comment by peeech** [comment]
cc @yegortimoshenko

File: overlays/holo-nixpkgs/holochain-rust/default.nix [hunk 1]
```diff
@@ -6,11 +6,11 @@ rustPlatform.buildRustPackage {
   src = fetchFromGitHub {
     owner = "holochain";
     repo = "holochain-rust";
-    rev = "v0.0.42-alpha5";
-    sha256 = "0x1vnhyq6ksjk2ci7c5kw6n1l8dwfz940sd4vzrcmzl7hhwvpd45";
+    rev = "3a88ae4977162b4310aae4dcef1673844ec76d84";
```
Comment (zo-el) [inline overlays/holo-nixpkgs/holochain-rust/default.nix]: @peeech, can we use the release tag `v0.0.45-alpha1` here instead of the commit hash?

Comment (peeech) [reply overlays/holo-nixpkgs/holochain-rust/default.nix]: yes. I'm force-pushing

Comment (peeech) [reply overlays/holo-nixpkgs/holochain-rust/default.nix]: Actually no because it will change commit hash and all our dependency pyramid will collapse

**Pull request merged**